frontend/src/components/Dashboard.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Header from "../commons/Header";
+import Buttons from "../commons/Buttons";
+import {useHistory} from "react-router-dom";
+import usePositioning from "../hooks/usePositioning";
+
+export default function Dashboard() {
+
+    const history = useHistory();
+
+    const [positionAvailable, latitude, longitude, errorMessage] = usePositioning();
+
+    return (
+        <PageLayout>
+            <Header headerText={"Ahoi 'username'"}/>
+            <PositionBlock>
+                <div>Current location: {positionAvailable ? <b>Available</b> : <b>Not Available</b>}</div>
```

Comment:
Better use a `section`, `div`s are just for styling. Use of `b` is dicouraged, style with css.

Reply:
But what is best practice for only having a part of a text as bold text?

Reply:
could be something as simple as `const Bold = styled.span`, using `b` is discouraged as it conveys no semantic meaning and we want to restrict html to semantics not styling